### ISSUE TITLE
Only mp4 videos are handled, m4v videos are ignored.

### ISSUE THREAD

**Issue by ryanbourneuk** [3sidedcube/ThunderCloud#167]
## Expected Behaviour

Both mp4 and m4v videos should be handled, as per how Android handles videos within Storm bundles.

## Current Behaviour

mp4 videos are handled correctly, but m4v videos are not.

## Possible Solution

Add an array defining the supported video file extensions, then check when attempting to open the video link that the extension is in the supported list.

## Steps to Reproduce (for bugs)

1. Within a Storm app, attempt to tap on a video that is of the m4v video type.
2. It does not load.
3. Attempt to tap on a video that is of the mp4 video type.
4. It does load.

## Context

This issue has occurred after the YouTube removal (a033c7f0bc731ab154faf6099a45e243f2045e57), where in VideoListItem.swift

```swift
	videoRow.parentNavigationController?.push(videos: videos)
```

was changed to instead be

```swift
        guard let video = videos.first(where: { $0.localeString == StormLanguageController.shared.currentLanguage }) ?? videos.first, let link = video.link else {
            return
        }
		
        parentNavigationController?.push(link: link)
```

This meant that the file extension checks were then being enforced, and m4v videos would no longer be presented.

## Your Environment

* ThunderCloud version: v1.3.1
* iOS / macOS version: n/a
* Link to your project, if public: n/a

**Comment by ryanbourneuk** [comment]
Fixed in #168.

**Comment by ryanbourneuk** [comment]
Resolved in the 1.3.2 release.
